src/pages/order/Cart/Cart.jsx

```diff
@@ -1,20 +1,69 @@
 import React, { Component } from 'react';
+import { CART_API, ERROR_MESSAGES } from '../../../config';
+import Loader from '../../../components/Loader/Loader';
 import Container from '../../../components/Container/Container';
 import CartTable from './CartTable';
 import TotalPriceTable from './TotalPriceTable';
 import BuyButtons from './BuyButtons';
 import './Cart.scss';
 
 class Cart extends Component {
+  constructor() {
+    super();
+    this.state = {
+      cartItems: [],
+      isLoading: true,
+    };
+  }
+
+  getCartData = async () => {
+    const accessToken = localStorage.getItem('token');
+
+    if (!accessToken) {
+      return this.setState({ isLoading: false });
+    }
```

Comment:
early return 굿!
setState를 return 하는 게 어색해서 분리하는 게 더 깔끔할 것 같습니다!
```suggestion
    if (!accessToken) {
       this.setState({ isLoading: false });
       return;
    }
```

Reply:
앗 그렇군요! 제안해주신 방법으로 수정했습니다!

https://github.com/wecode-bootcamp-korea/fullstack1-1st-BaseNote-frontend/pull/28/commits/284b2a0cec8d79cf8a15905b1aed5027e383265b